<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vpProjectEditor</class>
 <widget class="QDialog" name="vpProjectEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure - VisGUI View</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="datasetLabel">
     <property name="toolTip">
      <string>Path to the project imagery. This may be an image list file, a KWA index file, or a glob expression of image files.</string>
     </property>
     <property name="text">
      <string>&amp;Imagery</string>
     </property>
     <property name="buddy">
      <cstring>dataset</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="datasetLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLineEdit" name="dataset">
       <property name="toolTip">
        <string>Path to the project imagery. This may be an image list file, a KWA index file, or a glob expression of image files.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pickDataset">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../Icons/vpView.qrc">
         <normaloff>:/icons/16x16/browse</normaloff>:/icons/16x16/browse</iconset>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="tracksLabel">
     <property name="toolTip">
      <string>Path to tracks file.</string>
     </property>
     <property name="text">
      <string>&amp;Tracks</string>
     </property>
     <property name="buddy">
      <cstring>tracks</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="tracksLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLineEdit" name="tracks">
       <property name="toolTip">
        <string>Path to tracks file.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pickTracks">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../Icons/vpView.qrc">
         <normaloff>:/icons/16x16/browse</normaloff>:/icons/16x16/browse</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>448</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionDatasetPickFile">
   <property name="text">
    <string>Pick &amp;Image List / KWA...</string>
   </property>
  </action>
  <action name="actionDatasetPickDirectory">
   <property name="text">
    <string>Pick &amp;Directory...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../Icons/vpView.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>rejected()</signal>
   <receiver>vpProjectEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
